charts/README.md

```diff
@@ -1,10 +1,13 @@
 # RadonDB-MySQL
+
 RadonDB-MySQL is a open-source, cloud-native, High-Availability cluster solutions that is based on MySQL.
```

Comment:
`a open-source`--->`an open-source`
`solutions that is based on MySQL`--->`solutions  based on MySQL`